specs/eip4844/polynomial-commitments.md

```diff
@@ -46,13 +51,18 @@ This document specifies basic polynomial operations and KZG polynomial commitmen
 | `BLSFieldElement` | `uint256` | `x < BLS_MODULUS` |
 | `KZGCommitment` | `Bytes48` | Same as BLS standard "is valid pubkey" check but also allows `0x00..00` for point-at-infinity |
 | `KZGProof` | `Bytes48` | Same as for `KZGCommitment` |
+| `Polynomial` | `Vector[BLSFieldElement, FIELD_ELEMENTS_PER_BLOB]` | a polynomial in evaluation form |
```

Comment:
Add `Blob`; `Blob` only list of bytes
(This is Dankrad reviewing)

Reply:
This is reasonable to me, as it removes the need for non-crypto modules to know about field elements.

Reply:
This has been implemented, can mark as resolved